### PR TITLE
Starts new docs index UI

### PR DESCRIPTION
**Pull request by jmelberg-okta**
## Description:
- Updates UI to look closer to: [New Documentation Index](https://okta.invisionapp.com/share/2FBI90NN7#/screens/238045825)

screen shot 2017-08-24 at 11 24 54 am

- :art: Updates Quickstart spacing

screen shot 2017-08-22 at 5 25 58 pm

- :fire: Removes telephone number from Support dropdown
- :fire: Removes old code page
- :art: Updates quickstart icons

### Resolves:

* [OKTA-138486](https://oktainc.atlassian.net/browse/OKTA-138486)

### CODE REVIEW

**Comment by alexdahl-okta** [comment]
@jmelberg-okta Excited to see this! 👏 Let me know if I can do anything to help

**Comment by jmelberg-okta** [comment]
Hey @alexdahl-okta! This PR is mostly cosmetic changes. We are planning to add in the `code` icons soon, but in the meantime please feel free to leave feedback!

File: _source/_assets/js/quickstart.js [hunk 1]
```diff
@@ -276,4 +276,38 @@
...
+    { name: 'dotnet', label: '.NET' },
+    { name: 'php', label: 'PHP' }
+  ]
+
+  function renderCodeLinks() {
+    var ul = $('<ul>', {});
+    code.forEach(function (language) {
+      var li = $('<li>', {});
+      var span = $('<i>', {
+        class: `icon code-${language.name}-32`,
```
Comment (robertjd) [inline _source/_assets/js/quickstart.js]: These template literals are not supported in IE11, we get syntax errors and no links

File: _source/_assets/js/quickstart.js [hunk 2]
```diff
@@ -276,4 +276,38 @@
...
+    { name: 'angular', label: 'Angular' },
+    { name: 'react',label: 'React' },
+    { name: 'swift',  label: 'Swift' },
+    { name: 'nodejs', label: 'Node.js' },
+    { name: 'java', label: 'Java' },
+    { name: 'dotnet', label: '.NET' },
+    { name: 'php', label: 'PHP' }
+  ]
+
+  function renderCodeLinks() {
```
Comment (robertjd) [inline _source/_assets/js/quickstart.js]: Can we move this to a new JS file that deals with the docs home page specifically?

File: code/index.html [hunk 3]
```diff
@@ -232,6 +245,20 @@
     
 
     <div>
+      <a href="/code/nodejs/">
```
Comment (robertjd) [inline code/index.html]: Note: I left this comment in the wrong place.  I'm seeing this problem on the links that get rendered on the docs home page (not the /code page)

File: code/index.html [hunk 4]
```diff
@@ -141,6 +140,20 @@
     
 
     <div>
+      <a href="/code/angular/">
```
Comment (robertjd) [inline code/index.html]: I think we should just link directly to `code/javascript`.  The 302 feels unnecessary (and I get an interstitial page which tells me I'm redirecting, which also feels unnecessary)

**Comment by alexdahl-okta** [comment]
@jmelberg-okta: a few asks for the docs index hero:

- Can we make sure the entire tile is a link? (Not just the name.)
- Swift should be labeled as "iOS".
- The arrow on each tile should be right-facing, as it opens in the same tab. Probably `&rsaquo`;?
- I don't think we need the hover state.
- Can we add more padding to the bottom of the hero? Maybe ~30px?

Thanks!

**Comment by robertjd** [comment]
@jmelberg-okta please see outdated comments, I left the on the /code page which has been removed, but are still relevant

File: code/index.html (hunk 4 above)
Comment (alexdahl-okta) [inline code/index.html]: Agree that a direct link is better than the redirect. But ideally, we should have an Angular page to link to.

**Comment by jmelberg-okta** [comment]
@alexdahl-okta I was able to modify the page per your above comments - with the exception of *making the entire tile a link*. I'll have to investigate how to do this without breaking CSS.

**Pull request merged**